src/main/java/org/opentripplanner/standalone/config/buildconfig/NetexConfig.java

```diff
@@ -148,6 +148,13 @@ private static NetexFeedParameters.Builder mapFilePatternParameters(
             """
           )
           .asStringSet(original.ferryIdsNotAllowedForBicycle())
+      )
+      .withIgnoreFareFrame(
+        config
+          .of("ignoreFareFrame")
+          .since(V2_2)
```

Comment:
Since 2.2 has been released we need a new enum value for 2.3

Reply:
Updated